create-turbo/.eslintrc.js

```
@@ -1,6 +1,26 @@
 module.exports = {
+  root: true,
+  parser: "@typescript-eslint/parser",
+  env: { node: true, jest: true },
+  extends: [
+    "eslint:recommended",
+    "plugin:import/recommended",
+    "plugin:import/typescript",
+  ],
+  plugins: ["import"],
   rules: {
-    // we have an example where we need this
-    "no-undef": "off",
+    "no-empty": ["error", { allowEmptyCatch: true }],
```

allow empty catch blocks:
https://github.com/vercel/turborepo/blob/08bdd3c4845b51d2ca08a1460f71ec886e7e4742/create-turbo/src/git.ts#L7-L10